account-compression/sdk/src/generated/types/ChangeLogEvent.ts

```diff
@@ -32,20 +31,22 @@ export type ChangeLogEventRecord = {
  * @category enums
  * @category generated
  */
-export type ChangeLogEvent = beet.DataEnumKeyAsKind<ChangeLogEventRecord>;
+export type ChangeLogEvent = beet.DataEnumKeyAsKind<ChangeLogEventRecord>
 
-export const isChangeLogEventV1 = (x: ChangeLogEvent): x is ChangeLogEvent & { __kind: 'V1' } => x.__kind === 'V1';
+export const isChangeLogEventV1 = (
+  x: ChangeLogEvent
+): x is ChangeLogEvent & { __kind: 'V1' } => x.__kind === 'V1'
 
 /**
  * @category userTypes
  * @category generated
  */
 export const changeLogEventBeet = beet.dataEnum<ChangeLogEventRecord>([
-    [
-        'V1',
-        new beet.FixableBeetArgsStruct<ChangeLogEventRecord['V1']>(
-            [['fields', beet.tuple([changeLogEventV1Beet])]],
-            'ChangeLogEventRecord["V1"]',
-        ),
-    ],
-]) as beet.FixableBeet<ChangeLogEvent>;
+  [
+    'V1',
+    new beet.FixableBeetArgsStruct<ChangeLogEventRecord['V1']>(
+      [['fields', beet.tuple([changeLogEventV1Beet])]],
+      'ChangeLogEventRecord["V1"]'
+    ),
+  ],
+]) as beet.FixableBeet<ChangeLogEvent, ChangeLogEvent>
```

Comment:
Nit: Has the Solita autogenerated code been run through a different linter?  It seems so because afaik Solita generates with commas and semicolons.  I would recommend regenerating the Solita TS and not linting it, in order to keep it consistent with other generated Solita code and minimize changed files for the next PR.  (I do agree the 2-space indent belongs, and I think Solita does that by default).

Reply:
Yeah got it, I think that @ngundotra just didn't want to see all the lint changes as part of this PR, but @StanChe and I were looking into how to actually get the format that is checked in.

It looks like by default the prettierrc.yaml file is not used, so you have to do something like this:
 ```
 pnpm solita
 pnpm lint:fix
 pnpm prettier --config prettierrc.yaml  --write idl/spl_account_compression.json
```

I still see some slight differences in the output and had some error messages in the output, but it did work. 
 @StanChe might have gotten cleaner results but wasn't sure of his local setup.
 
Having said all this, I think the bigger issue is the missing canopy issue described below in @StanChe 's comment.  This could be a blocker for the PR and I think @StanChe is looking into a solution to initialize the canopy on a tree that uses one.